lib/IDE/REPLCodeCompletion.cpp

```diff
@@ -194,6 +194,10 @@ doCodeCompletion(SourceFile &SF, StringRef EnteredCode, unsigned *BufferID,
                  CodeCompletionCallbacksFactory *CompletionCallbacksFactory) {
   // Temporarily disable printing the diagnostics.
   ASTContext &Ctx = SF.getASTContext();
+
+  // Register the request evaluator.
+  registerIDERequestFunctions(Ctx.evaluator);
```

Comment:
I think you only want to do this once when the ASTContext is being set up, not every time.

Reply:
Cool, I'll change that, thanks!

Reply:
Done.